csharp/test/Vista.SDK.Tests/GmodTests.cs

```diff
@@ -46,21 +46,51 @@ public void Test_Gmod_Properties(VisVersion visVersion)
         var gmod = vis.GetGmod(visVersion);
         Assert.NotNull(gmod);
 
-        var minLength = gmod.MinBy(n => n.Code.Length);
-        var maxLength = gmod.MaxBy(n => n.Code.Length);
-        Assert.NotNull(minLength);
-        Assert.NotNull(maxLength);
-        Assert.Equal(2, minLength.Code.Length);
-        Assert.Equal("VE", minLength.Code);
-        Assert.Equal(10, maxLength.Code.Length);
+        GmodNode? minLengthNode = null;
+        int currentMinLength = int.MaxValue;
 
+        GmodNode? maxLengthNode = null;
+        int currentMaxLength = 0;
+        int nodeCount = 0;
+
+        foreach (var node in gmod)
+        {
+            nodeCount++;
+            string code = node.Code;
+            int len = code.Length;
+
+            if (minLengthNode is null || len < currentMinLength)
+            {
+                currentMinLength = len;
+                minLengthNode = node;
+            }
```

Comment:
Include same lexiographical test for min length. We assume Gmod is ordered, but for the sake of the test, it is good practice.